src/graphql/main/schema.graphql

```diff
@@ -253,8 +253,8 @@ type Coordinates {
 }
 
 type Currency {
-  code: String!
   flag: String!
+  id: String!
```

Comment:
`id` should be type `ID`. it's an alias for string expect I think it means it should be unique.

Reply:
which is how it's automatically cached by the client afaiu